tests/unit/test_main.py

```diff
@@ -318,6 +387,249 @@ def test_disable_interfaces(self, mock_save_status):
         self.assertEqual(mock_interface_1.disable.call_count, 0)
         self.assertEqual(mock_interface_2.disable.call_count, 0)
 
+    def test_get_interface_metadata(self):
+        """Test get_interface_metada."""
+        interface_id = '00:00:00:00:00:00:00:01:1'
+        dpid = '00:00:00:00:00:00:00:01'
+        msg_error_switch = "Switch not found"
+        msg_error_interface = "Interface not found"
+        mock_switch = get_switch_mock(dpid)
+        mock_interface = create_autospec(Interface)
```

Comment:
Maybe we can use the `get_interface_mock` method of kytos core helpers lib.

Reply:
Implemented. Thanks a lot for all suggestions.